runtime/builtin_types.go

```diff
@@ -559,6 +560,42 @@ func builtinRange(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException)
 	return ListType.Call(f, []*Object{r}, nil)
 }
 
+func builtinRawInput(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	if len(args) > 1 {
+		msg := fmt.Sprintf("[raw_]input expcted at most 1 arguments, got %d", len(args))
+		return nil, f.RaiseType(TypeErrorType, msg)
+	}
+
+	fin := os.Stdin
+	fout := os.Stdout
```

Comment:
Rather than writing to Stdout directly, you could use the Print() function: `Print(f, args, false)`